./app.go:70:37: undefined: VERSION

Since version 1.1.0 `go build` failed.
```bash
==> Starting prepare()...
github.com/fatih/color (download)
cd .
git clone https://github.com/fatih/color /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/fatih/color
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/fatih/color
git submodule update --init --recursive
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/fatih/color
git show-ref
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/fatih/color
git submodule update --init --recursive
WORK=/tmp/go-build839796451
mkdir -p $WORK/b001/
mkdir -p /home/k0ste/sandbox/AUR/ip2geo/src/gopath/pkg/linux_amd64/github.com/fatih/
cp /home/k0ste/.cache/go-build/d6/d63af25eb5f99935f2480a8c7573ab5fd4a7a35d3e110ed6b2f7637a5b6c9f5e-d /home/k0ste/sandbox/AUR/ip2geo/src/gopath/pkg/linux_amd64/github.com/fatih/color.a
rm -r $WORK/b001/
github.com/jinzhu/configor (download)
cd .
git clone https://github.com/jinzhu/configor /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/jinzhu/configor
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/jinzhu/configor
git submodule update --init --recursive
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/jinzhu/configor
git show-ref
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/jinzhu/configor
git submodule update --init --recursive
github.com/BurntSushi/toml (download)
cd .
git clone https://github.com/BurntSushi/toml /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/BurntSushi/toml
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/BurntSushi/toml
git submodule update --init --recursive
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/BurntSushi/toml
git show-ref
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/BurntSushi/toml
git submodule update --init --recursive
Fetching https://gopkg.in/yaml.v2?go-get=1
Parsing meta tags from https://gopkg.in/yaml.v2?go-get=1 (status code 200)
get "gopkg.in/yaml.v2": found meta tag get.metaImport{Prefix:"gopkg.in/yaml.v2", VCS:"git", RepoRoot:"https://gopkg.in/yaml.v2"} at https://gopkg.in/yaml.v2?go-get=1
gopkg.in/yaml.v2 (download)
cd .
git clone https://gopkg.in/yaml.v2 /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/gopkg.in/yaml.v2
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/gopkg.in/yaml.v2
git submodule update --init --recursive
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/gopkg.in/yaml.v2
git show-ref
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/gopkg.in/yaml.v2
git submodule update --init --recursive
WORK=/tmp/go-build227348624
github.com/BurntSushi/toml
mkdir -p $WORK/b036/
cat >$WORK/b036/importcfg << 'EOF' # internal
# import config
packagefile bufio=/usr/lib/go/pkg/linux_amd64/bufio.a
packagefile encoding=/usr/lib/go/pkg/linux_amd64/encoding.a
packagefile errors=/usr/lib/go/pkg/linux_amd64/errors.a
packagefile fmt=/usr/lib/go/pkg/linux_amd64/fmt.a
packagefile io=/usr/lib/go/pkg/linux_amd64/io.a
packagefile io/ioutil=/usr/lib/go/pkg/linux_amd64/io/ioutil.a
packagefile math=/usr/lib/go/pkg/linux_amd64/math.a
packagefile reflect=/usr/lib/go/pkg/linux_amd64/reflect.a
packagefile sort=/usr/lib/go/pkg/linux_amd64/sort.a
packagefile strconv=/usr/lib/go/pkg/linux_amd64/strconv.a
packagefile strings=/usr/lib/go/pkg/linux_amd64/strings.a
packagefile sync=/usr/lib/go/pkg/linux_amd64/sync.a
packagefile time=/usr/lib/go/pkg/linux_amd64/time.a
packagefile unicode=/usr/lib/go/pkg/linux_amd64/unicode.a
packagefile unicode/utf8=/usr/lib/go/pkg/linux_amd64/unicode/utf8.a
EOF
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/BurntSushi/toml
/usr/lib/go/pkg/tool/linux_amd64/compile -o $WORK/b036/_pkg_.a -trimpath $WORK/b036 -p github.com/BurntSushi/toml -complete -buildid cRaxoTmtHcCsdZBCSzoo/cRaxoTmtHcCsdZBCSzoo -goversion go1.12.5 -D "" -importcfg $WORK/b036/importcfg -pack -c=4 ./decode.go ./decode_meta.go ./doc.go ./encode.go ./encoding_types.go ./lex.go ./parse.go ./type_check.go ./type_fields.go
gopkg.in/yaml.v2
mkdir -p $WORK/b040/
cat >$WORK/b040/importcfg << 'EOF' # internal
# import config
packagefile bytes=/usr/lib/go/pkg/linux_amd64/bytes.a
packagefile encoding=/usr/lib/go/pkg/linux_amd64/encoding.a
packagefile encoding/base64=/usr/lib/go/pkg/linux_amd64/encoding/base64.a
packagefile errors=/usr/lib/go/pkg/linux_amd64/errors.a
packagefile fmt=/usr/lib/go/pkg/linux_amd64/fmt.a
packagefile io=/usr/lib/go/pkg/linux_amd64/io.a
packagefile math=/usr/lib/go/pkg/linux_amd64/math.a
packagefile reflect=/usr/lib/go/pkg/linux_amd64/reflect.a
packagefile regexp=/usr/lib/go/pkg/linux_amd64/regexp.a
packagefile sort=/usr/lib/go/pkg/linux_amd64/sort.a
packagefile strconv=/usr/lib/go/pkg/linux_amd64/strconv.a
packagefile strings=/usr/lib/go/pkg/linux_amd64/strings.a
packagefile sync=/usr/lib/go/pkg/linux_amd64/sync.a
packagefile time=/usr/lib/go/pkg/linux_amd64/time.a
packagefile unicode=/usr/lib/go/pkg/linux_amd64/unicode.a
packagefile unicode/utf8=/usr/lib/go/pkg/linux_amd64/unicode/utf8.a
EOF
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/gopkg.in/yaml.v2
/usr/lib/go/pkg/tool/linux_amd64/compile -o $WORK/b040/_pkg_.a -trimpath $WORK/b040 -p gopkg.in/yaml.v2 -complete -buildid c60_lbL5B20vI4O4h6ej/c60_lbL5B20vI4O4h6ej -goversion go1.12.5 -D "" -importcfg $WORK/b040/importcfg -pack -c=4 ./apic.go ./decode.go ./emitterc.go ./encode.go ./parserc.go ./readerc.go ./resolve.go ./scannerc.go ./sorter.go ./writerc.go ./yaml.go ./yamlh.go ./yamlprivateh.go
/usr/lib/go/pkg/tool/linux_amd64/buildid -w $WORK/b036/_pkg_.a # internal
cp $WORK/b036/_pkg_.a /home/k0ste/.cache/go-build/c9/c9217b9d48235d75f15d39e97abac15eea56258ae76cbad0275dbce56be9e841-d # internal
/usr/lib/go/pkg/tool/linux_amd64/buildid -w $WORK/b040/_pkg_.a # internal
cp $WORK/b040/_pkg_.a /home/k0ste/.cache/go-build/d6/d6e81b4b8c5cd024c9bb139d85d8f72567fd65c66e830dc8e532348f102fb459-d # internal
github.com/jinzhu/configor
mkdir -p $WORK/b001/
cat >$WORK/b001/importcfg << 'EOF' # internal
# import config
packagefile encoding/json=/usr/lib/go/pkg/linux_amd64/encoding/json.a
packagefile errors=/usr/lib/go/pkg/linux_amd64/errors.a
packagefile fmt=/usr/lib/go/pkg/linux_amd64/fmt.a
packagefile github.com/BurntSushi/toml=$WORK/b036/_pkg_.a
packagefile gopkg.in/yaml.v2=$WORK/b040/_pkg_.a
packagefile io=/usr/lib/go/pkg/linux_amd64/io.a
packagefile io/ioutil=/usr/lib/go/pkg/linux_amd64/io/ioutil.a
packagefile os=/usr/lib/go/pkg/linux_amd64/os.a
packagefile path=/usr/lib/go/pkg/linux_amd64/path.a
packagefile reflect=/usr/lib/go/pkg/linux_amd64/reflect.a
packagefile regexp=/usr/lib/go/pkg/linux_amd64/regexp.a
packagefile strings=/usr/lib/go/pkg/linux_amd64/strings.a
packagefile time=/usr/lib/go/pkg/linux_amd64/time.a
EOF
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/github.com/jinzhu/configor
/usr/lib/go/pkg/tool/linux_amd64/compile -o $WORK/b001/_pkg_.a -trimpath $WORK/b001 -p github.com/jinzhu/configor -complete -buildid MtnCkr4buxtKVkpUZ7Fm/MtnCkr4buxtKVkpUZ7Fm -goversion go1.12.5 -D "" -importcfg $WORK/b001/importcfg -pack -c=4 ./configor.go ./utils.go
/usr/lib/go/pkg/tool/linux_amd64/buildid -w $WORK/b001/_pkg_.a # internal
cp $WORK/b001/_pkg_.a /home/k0ste/.cache/go-build/03/033a0cbd1ec1f6224c075eb946af9f2be7e65fd34aee43434ff445403882b6a0-d # internal
mkdir -p /home/k0ste/sandbox/AUR/ip2geo/src/gopath/pkg/linux_amd64/github.com/jinzhu/
cp $WORK/b001/_pkg_.a /home/k0ste/sandbox/AUR/ip2geo/src/gopath/pkg/linux_amd64/github.com/jinzhu/configor.a
rm -r $WORK/b001/
Fetching https://golang.org/x/net/html/charset?go-get=1
Parsing meta tags from https://golang.org/x/net/html/charset?go-get=1 (status code 200)
get "golang.org/x/net/html/charset": found meta tag get.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/html/charset?go-get=1
get "golang.org/x/net/html/charset": verifying non-authoritative meta tag
Fetching https://golang.org/x/net?go-get=1
Parsing meta tags from https://golang.org/x/net?go-get=1 (status code 200)
golang.org/x/net (download)
cd .
git clone https://go.googlesource.com/net /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/golang.org/x/net
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/golang.org/x/net
git submodule update --init --recursive
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/golang.org/x/net
git show-ref
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/golang.org/x/net
git submodule update --init --recursive
Fetching https://golang.org/x/text/encoding?go-get=1
Parsing meta tags from https://golang.org/x/text/encoding?go-get=1 (status code 200)
get "golang.org/x/text/encoding": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at https://golang.org/x/text/encoding?go-get=1
get "golang.org/x/text/encoding": verifying non-authoritative meta tag
Fetching https://golang.org/x/text?go-get=1
Parsing meta tags from https://golang.org/x/text?go-get=1 (status code 200)
golang.org/x/text (download)
cd .
git clone https://go.googlesource.com/text /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/golang.org/x/text
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/golang.org/x/text
git submodule update --init --recursive
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/golang.org/x/text
git show-ref
cd /home/k0ste/sandbox/AUR/ip2geo/src/gopath/src/golang.org/x/text
git submodule update --init --recursive
Fetching https://golang.org/x/text/transform?go-get=1
Parsing meta tags from https://golang.org/x/text/transform?go-get=1 (status code 200)
get "golang.org/x/text/transform": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at https://golang.org/x/text/transform?go-get=1
get "golang.org/x/text/transform": verifying non-authoritative meta tag
Fetching https://golang.org/x/text/encoding/charmap?go-get=1
Parsing meta tags from https://golang.org/x/text/encoding/charmap?go-get=1 (status code 200)
get "golang.org/x/text/encoding/charmap": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at https://golang.org/x/text/encoding/charmap?go-get=1
get "golang.org/x/text/encoding/charmap": verifying non-authoritative meta tag
Fetching https://golang.org/x/text/encoding/htmlindex?go-get=1
Parsing meta tags from https://golang.org/x/text/encoding/htmlindex?go-get=1 (status code 200)
get "golang.org/x/text/encoding/htmlindex": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at https://golang.org/x/text/encoding/htmlindex?go-get=1
get "golang.org/x/text/encoding/htmlindex": verifying non-authoritative meta tag
WORK=/tmp/go-build336894592
mkdir -p $WORK/b001/
mkdir -p /home/k0ste/sandbox/AUR/ip2geo/src/gopath/pkg/linux_amd64/golang.org/x/net/html/
cp /home/k0ste/.cache/go-build/85/853429d57f6691abec2af2f7b2eaefc60a5451d1ed89022b6ab03c78cb3dc965-d /home/k0ste/sandbox/AUR/ip2geo/src/gopath/pkg/linux_amd64/golang.org/x/net/html/charset.a
rm -r $WORK/b001/
==> Starting build()...
# github.com/m-messiah/ip2geo
./app.go:70:37: undefined: VERSION
==> ERROR: A failure occurred in build().
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,7 +9,7 @@ before_install:
   - cpanm --local-lib=~/perl5 --quiet local::lib
   - eval $(perl -I ~/perl5/lib/perl5/ -Mlocal::lib)
   - cpanm --local-lib=~/perl5 --quiet --notest Test::Nginx::Socket
-  - echo "const VERSION = \"${TRAVIS_BRANCH}\"" >> version.go
+  - sed -i 's/const VERSION = .*/const VERSION = "'${TRAVIS_BRANCH}'"/' version.go
 script:
   - go build -o ip2geo
   - ./ip2geo -version

--- a/version.go
+++ b/version.go
@@ -1,1 +1,4 @@
 package main
+
+// VERSION for `-version` flag output. Equals to git tag in releases.
+const VERSION = "unversioned"

